pkg/apis/admissionregistration/types.go

```diff
@@ -235,6 +235,13 @@ type Webhook struct {
 	// Default to the empty LabelSelector, which matches everything.
 	// +optional
 	NamespaceSelector *metav1.LabelSelector
+
+	// DryRunnable defines whether this webhook will correctly handle dryRun requests.
+	// If false, any dryRun requests to resources/subresources this webhook applies to
+	// will be completely rejected and the webhook will not be called.
+	// Defaults to false.
+	// +optional
+	DryRunnable *bool
```

Comment:
I recommend pulling out your API change and the one admission plugin that uses it into a separate pull so you can get 90% of the way without an api-review.